scripts/vim-patch.sh

```diff
@@ -5,6 +5,12 @@ set -u
 # Use privileged mode, which e.g. skips using CDPATH.
 set -p
 
+# Ensure that the user has a bash that supports -A
+if [ "${BASH_VERSION::1}" -lt 4  ]; then
```

Comment:
Should use `BASH_VERSINFO[0]` ("The major version number (the release).", see `bash(1)`).
Otherwise it fails for bash 10 in some years.
Have not checked when `BASH_VERSINFO` was added though..

Reply:
https://github.com/bminor/bash/blob/master/CHANGES#L9370
Versinfo was added in bash 2.0, so 1996. Fixed now.